eo-parser/src/main/resources/log4j.properties

```diff
@@ -1,7 +1,7 @@
 log4j.rootLogger=WARN, CONSOLE
 
 log4j.appender.CONSOLE=org.apache.log4j.ConsoleAppender
-log4j.appender.CONSOLE.layout=org.apache.log4j.PatternLayout
-log4j.appender.CONSOLE.layout.ConversionPattern=BUILD %c: %m%n
+log4j.appender.CONSOLE.layout=com.jcabi.log.MulticolorLayout
+log4j.appender.CONSOLE.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss} [%color{%p}] %c: %m%n
```

Comment:
@volodya-lombrozo same here, why year/month/day?

Reply:
@yegor256 fixed. I've left only `HH:mm:ss`. If we leave only `mm:ss` - it will confuse, because looks like `HH:mm`